modules/components/pages/redpanda/about.adoc

```diff
@@ -321,7 +386,97 @@ The token to use for a single session's OAUTHBEARER authentication.
 
 === `sasl[].extensions`
 
-Key/value pairs to add to OAUTHBEARER authentication requests.
+Key/value pairs to add to `OAUTHBEARER` authentication requests.
+
+
+=== `sasl[].aws`
+
+AWS specific fields for when the `mechanism` is set to `AWS_MSK_IAM`.
+
+*Type*: `object`
+
+=== `sasl[].aws.region`
+
+The AWS region to target.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].aws.endpoint`
+
+Specify a custom endpoint for the AWS API.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].aws.credentials`
+
+Manually configure the AWS credentials to use (optional). For more information, see the xref:guides:cloud/aws.adoc[Amazon Web Services guide].
```

Comment:
```suggestion
Manually configure the AWS credentials to use (optional). For more information, see the xref:guides:cloud/aws.adoc[Amazon Web Services guide^].
```

Reply:
This is an internal RPCN guide, so users are not being taken outside of the docs site.

Reply:
I don't understand this. Above on the same page, there's an [external link](https://deploy-preview-64--redpanda-connect.netlify.app/redpanda-connect/components/redpanda/about/#status_topic) that's correctly formatted. This would just do the same.

Reply:
OK. Probably me not explaining this very well. This is not an external link. There is an RPCN Amazon Web Services guide for users to set up credentials in pipelines. This what the link opens.

Reply:
👌